metaci/plan/models.py

```diff
@@ -191,9 +201,19 @@ def check_github_event(self, event, payload):
             and self.trigger == "status"
             and payload["state"] == "success"
         ):
-            if not re.match(self.regex, payload["context"]):
+            if not re.match(self.commit_status_regex, payload["context"]):
```

Comment:
This will break if commit_status_regex is None; see my prior comment re validation.